src/CoreWlan/Enums.cs

```diff
@@ -136,7 +137,11 @@ public enum CWEventType : long {
 		LinkQualityDidChange = 6,
 		ModeDidChange = 7,
 		ScanCacheUpdated = 8,
+
+		[Deprecated (PlatformName.MacOSX, 10,16)]
 		VirtualInterfaceStateChanged = 9,
+
+		[Deprecated (PlatformName.MacOSX, 10,16)]
 		RangingReportEvent = 10,
 		Unknown = long.MaxValue
```

Comment:
```suggestion
		Unknown = long.MaxValue,
```

Reply:
yeah, but not part of the changes, IOW don't mix styles changes with actual new code / bug fixes :)